klaas/scripts/train_separation_model.py

```diff
@@ -14,6 +14,52 @@
 from ..feature_generation import feature_generation
 
 
+logging.basicConfig()
+log = logging.getLogger()
+
+
+def read_and_sample_data(path, key, columns_to_read, n_sample):
+    df = read_data(
+        file_path=path,
+        key=key,
+        columns=columns_to_read,
+    )
+
+    if n_sample is not None:
+        if n_sample > len(df):
+            log.error(
+                'number of sampled events {} must be smaller than number events in file {} ({})'
+                .format(n_sample, path, len(df))
+            )
+            raise ValueError
+        log.info('Randomly sample {} events'.format(n_sample))
+        df = df.sample(n_sample)
+    return df
+
+
+def read_config(configuration_path):
```

Comment:
Same as above

Reply:
its kind of specific. I guess we can think about it in a separate PR